actions/workflows/st2_prep_release.yaml

```diff
@@ -125,6 +125,16 @@ st2cd.st2_prep_release:
                 local_repo: <% 'st2_dockerfiles_' + $.local_repo_sfx %>
                 hosts: <% $.host %>
                 cwd: <% $.cwd %>
+            on-success:
+                - wait_for_community_docker_images
+        wait_for_community_docker_image:
+            action: st2cd.wait_for_docker_image
+            input:
+                org: <% $.fork %>
+                image: st2workflowengine
+                version: <% $.version %>
+                tries: 45
+                default: 60
```

Comment:
What is the `default` parameter and his responsibility for this task?
Can't find it in meta

Reply:
If it's `delay`, then waiting for `45tries` x `60secs` which is 45 mins looks like too much.
Let's cap the upper limit to `10mins` or `15mins`. 

According to the `st2-dockerfiles` build, images are usually built in 3 mins, giving 7 more mins should be OK for uploading them to Docker Hub.